test/clojure/nrepl/core_test.clj

```diff
@@ -358,7 +358,7 @@
                                                        (def halted? false)))})]
     (Thread/sleep 100)
     (is (= #{"done"} (-> session (message {:op :interrupt}) first :status set)))
-    (is (= #{"done" "interrupted"} (-> resp combine-responses :status)))
+    (is (= #{} (reduce disj #{"done" "interrupted"} (-> resp combine-responses :status))))
```

Comment:
What can be the extra status?

Reply:
An `eval-error` managed to sneak in. (Because before stopping I first interrupt the thread)